### DISCUSSION TITLE
What is the smallest subatomic particle known to physics?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#215] [Galaxy Trivia] [answered]
**Hint:** ⬆️⬇️⬅️➡️ strange, and charm

**Comment by Faiqlatheef** [comment, marked as answer]
The smallest subatomic particle known to physics is the quark. Quarks are elementary particles and a fundamental constituent of matter. They are the smallest known particles that make up protons and neutrons, which in turn make up the nucleus of atoms. Quarks are believed to have no size, and they are always found in combination with other quarks in particles such as protons and neutrons. There are six known types of quarks: up, down, charm, strange, top, and bottom.
